addons/vagrant/inventory/group_vars/all/packetfence_install__api_calls.yml

```diff
@@ -0,0 +1,54 @@
+---
+packetfence_install__config_base_guests_admin_registration:
+  access_duration_choices: '10m,20m'
+packetfence_install__config_base_general:
+  hostname: pf
+  domain: example.lan
+  dhcpservers: 10.0.0.1,192.168.122.1
+  timezone: "{{ ansible_date_time['tz'] }}"
+packetfence_install__config_base_alerting:
+  smtpserver: localhost
+  fromaddr: "{{ packetfence_install__config_base_general['hostname'] }}@{{ packetfence_install__config_base_general['domain'] }}"
+  emailaddr: "alerting@{{ packetfence_install__config_base_general['domain'] }}"
+  smtp_port: 25
+  smtp_verifyssl: disabled
+  subjectprefix: PF UNDER TEST
+  smtp_timeout: 30
+packetfence_install__config_base_advanced:
+  hash_passwords: ntlm
+
+packetfence_install__api_calls:
+  - route: "{{ packetfence_install__api_config_endpoints['base'] }}/guests_admin_registration"
+    method: PATCH
+    body:
+      id: guests_admin_registration
+      access_duration_choices: "{{ packetfence_install__config_base_guests_admin_registration['access_duration_choices'] }}"
+  - route: "{{ packetfence_install__api_config_endpoints['base'] }}/general"
+    method: PATCH
+    body:
+      id: general
+      dhcpservers: "{{ packetfence_install__config_base_general['dhcpservers'] }}"
+      domain: "{{ packetfence_install__config_base_general['domain'] }}"
+      hostname: "{{ packetfence_install__config_base_general['hostname'] }}"
+      timezone: "{{ packetfence_install__config_base_general['timezone'] }}"
+  - route: "{{ packetfence_install__api_config_endpoints['base'] }}/alerting"
+    method: PATCH
+    body:
+      id: alerting
+      smtpserver: "{{ packetfence_install__config_base_alerting['smtpserver'] }}"
+      fromaddr: "{{ packetfence_install__config_base_alerting['fromaddr'] }}"
+      emailaddr: "{{ packetfence_install__config_base_alerting['emailaddr'] }}"
+      smtp_port: "{{ packetfence_install__config_base_alerting['smtp_port'] }}"
+      smtp_verifyssl: "{{ packetfence_install__config_base_alerting['smtp_verifyssl'] }}"
+      subjectprefix: "{{ packetfence_install__config_base_alerting['subjectprefix'] }}"
+      smtp_timeout: "{{ packetfence_install__config_base_alerting['smtp_timeout'] }}"
+  # - route: "{{ packetfence_install__api_config_endpoints['fingerbank_setting'] }}/upstream"
+  #   method: PATCH
+  #   body:
+  #     id: upstream
+  #     api_key: "{{ packetfence_install__config_fingerbank_setting_upstream['api_key'] }}"
+  - route: "{{ packetfence_install__api_config_endpoints['base'] }}/advanced"
+    method: PATCH
+    body:
+      id: advanced
+      hash_passwords: "{{ packetfence_install__config_base_advanced['hash_passwords'] }}"
```

Comment:
Where are those calls used?

Reply:
All in the [ansible-packetfence](https://github.com/inverse-inc/ansible-packetfence) repo I guess.

Reply:
@cgx,

You're right. `ansible-packetfence` is an Ansible collection that can't be install by Vagrant for now but it will the case in Ansible 2.9 according to https://github.com/ansible/ansible/pull/57106.

Reply:
Before Ansible 2.9 is released, I will install by hand `ansible-packetfence` collection on your CI server.

After Ansible 2.9 is released, I will update your CI server to Ansible 2.9 and update `requirements.yml` in order to install automatically `ansible-packetfence` when we do a `vagrant up`.